imageprocessor-api/src/test/java/pl/edu/uj/kimage/api/ApiTest.java

```diff
@@ -0,0 +1,39 @@
+package pl.edu.uj.kimage.api;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ApiTest {
```

Comment:
Can we write some kind of tests whether all the model class serializes correctly?